README.md

```diff
@@ -41,5 +48,26 @@ Finally, `tocrify` itself can be installed via `pip`:
 (env) $ pip install .
 ```
 
+## Invocation
+`tocrify` comes with a help message explaining its usage:
+```console
+(env) $ tocrify --help
+Usage: tocrify [OPTIONS] METS
+
+  METS: Input METS XML
+
+Options:
+  -o, --out-dir PATH         Directory for storing the updated OCR files
```

Comment:
I would like to see a hint that the directory has to exist!

Reply:
@wrznr Could also do

```python
if not os.path.exists(out_dir):
  makedirs(out_dir)
```

before writing out_filename

Reply:
@kba Sorry, I don't like/want my software to create directories.